src/app/space/create/deployments/apps/deployment-status-icon.component.ts

```diff
@@ -23,38 +23,13 @@ const STAT_THRESHOLD = .6;
   styleUrls: ['./deployment-status-icon.component.less']
 })
 export class DeploymentStatusIconComponent implements OnDestroy, OnInit {
-  @Input() cpuDataStream: Observable<CpuStat>;
 
-  iconClass: String;
-  toolTip: String;
-  subscriptions: Array<Subscription> = [];
+  @Input() iconClass: String;
+  @Input() toolTip: String;
 
-  constructor() {
-    this.iconClass = ICON_OK;
-    this.toolTip = MSG_OK;
-  }
+  constructor() { }
 
-  ngOnDestroy(): void {
-    this.subscriptions.forEach(sub => sub.unsubscribe());
-  }
+  ngOnDestroy(): void { }
 
-  ngOnInit(): void {
-    this.subscriptions.push(this.cpuDataStream.subscribe((stat) => {
-      this.changeStatus(stat);
-    }));
-  }
-
-  changeStatus(stat: CpuStat) {
-    this.iconClass = ICON_OK;
-    this.toolTip = MSG_OK;
-    if (stat.used / stat.quota > STAT_THRESHOLD) {
-      this.iconClass = ICON_WARN;
-      this.toolTip = 'CPU usage is approaching or at capacity.';
-    }
-
-    if (stat.used > stat.quota) {
-      this.iconClass = ICON_ERR;
-      this.toolTip = 'CPU usage has exceeded capacity.';
-    }
-  }
+  ngOnInit(): void { }
```

Comment:
May as well remove the constructor, ngOnDestroy, and ngOnInit if they're just going to be empty for this very simple component. If they're ever needed in the future then they can be re-added then.